src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Deployment/AzureADDeploymentSource.cs

```diff
@@ -25,10 +26,11 @@ public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlan
 
             var settings = await _azureADService.GetSettingsAsync();
 
-            result.Steps.Add(new JObject(
-                new JProperty("name", "AzureAD"),
-                new JProperty("AzureAD", JObject.FromObject(settings))
-            ));
+            var obj = new JObject(new JProperty("name", nameof(AzureADSettings)));
```

Comment:
```suggestion
            var step = new JObject(new JProperty("name", nameof(AzureADSettings)));
```

Reply:
I remember this will conflict with the parameter `step`